test/SparqlAntlrParserTest.cpp

```diff
@@ -758,9 +756,7 @@ TEST(SparqlParser, LimitOffsetClause) {
   {
     string input = "LIMIT20";
 
-    // parse* catches antlr4::ParseCancellationException and throws a
-    // std::runtime_error so this has to be checked instead.
-    EXPECT_THROW(parseLimitOffsetClause(input), std::runtime_error);
+    EXPECT_THROW(parseLimitOffsetClause(input), ParseException);
```

Comment:
Maybe, while you're at it, 
you can clean up the cases above, 
they are rather redundant:

```
expectLimitOffset("TEXTLIMIT 43 Limit 19", 19, 43, 0);
```
instead of lines 748 through 754.